scripts/srpm/fapolicy-analyzer.spec

```diff
@@ -229,6 +247,7 @@ alias python3=%{venv_py3}
 %endif
 
 python3 setup.py compile_catalog -f
+python3 setup.py build_help
```

Comment:
This cannot run here since we cannot rely on a network connection during hte RPM build.

Instead the compiled documentation archive should be added as a SourceX file and unpacked to the correct filesystem location during the RPM build.